### PR TITLE
Use FreeBSD_ports and packaging to find deps

### PR DESCRIPTION
**Pull request by swills**

### CODE REVIEW

**Review by tykling — APPROVED** [review]
works for me! :+1:

but it did not add the install_depends from setup.py

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,5 @@
 docutils>=0.12
 spdx-lookup>=0.3.0
+packaging
+setuptools
+FreeBSD-ports>=0.0.9
```
Comment (koobs) [inline requirements.txt:5]: Yep per @tykling these should go in 

`setup.py: install_requires = [`

**Review by koobs — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (koobs) [reply requirements.txt:5]: We should drop requirements.txt too, replacing the following line in README.md:

`$ pip install -r requirements.txt`

with

`$pip install pytoport`

**Comment by swills** [comment]
Could we do that in a separate change?

**Comment by koobs** [comment]
> 
> 
> Could we do that in a separate change?

Sure, just duplicate the additions to requirements.txt into setup.py (such that it can be pip installed), and we can remove the requirements.txt and README.md in a separate change.

Reminds me I should also try to obtain access to the PyPI account creds for pytoport. Will try to sort that out next week

**Comment by swills** [comment]
Done

**Review by koobs — APPROVED** [review]

File: pytoport/__init__.py [hunk 2]
```diff
@@ -137,15 +164,19 @@ def get_minimum(data):
 
     return "%s+ # %s" % (ver, ", ".join(others))
 
+def gen_dep(pkg):
+    ports = FreeBSD_ports()
+    portpath = ports.find_portdir("py36-" + pkg)
```
Comment (tykling) [inline pytoport/__init__.py]: is py36 here right?

Comment (swills) [reply pytoport/__init__.py]: No. This probably needs something to find the correct version of python but for now I'll just change it and hope to do that in a latter commit.

**Comment by swills** [comment]
Should I merge?

**Pull request merged**